pkg/kopia/repository/create.go

```diff
@@ -107,3 +114,24 @@ func setCustomMaintenanceOwner(
 	format.Log(pod, container, stderr)
 	return err
 }
+
+func getLocationAndCredsFromMountPath(
+	cli kubernetes.Interface,
+	namespace,
+	pod,
+	container string,
+) (map[string]string, map[string]string, error) {
+	fr := kube.NewPodFileReader(cli, pod, namespace, container)
+	loc, err := fr.ReadDir(context.Background(), LocationSecretMountPath)
```

Comment:
Instead of creating new context here, can we get that from caller as a function argument?